security/backends/elasticsearch/tests.py

```diff
@@ -3,6 +3,7 @@
 from django.test.utils import override_settings
 
 from .models import CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog, InputRequestLog, OutputRequestLog
+from .models import PartitionedLog
```

Comment:
why is there another import line instead of an addition to the one above?

Reply:
line length would be too long

Reply:
why not use parentheses instead? they are used elsewhere int he project

Reply:
Ok, but let's incorporate some linting, it's hard to search in project what conventions should be used.